README.md

```diff
@@ -23,7 +23,7 @@ A) Clone repo and checkout branch, go to mdcb folder
 ```
 git clone https://github.com/TykTechnologies/tyk-pro-docker-demo
 cd tyk-pro-docker-demo/
-git checkout mdcb
+git checkout improve-mdcb
```

Comment:
this is wrong rightt?  We want to still checckoutt `mdcb` after the merge is complete?